app/presenters/return-requirements/check-your-answers.presenter.js

```diff
@@ -4,16 +4,38 @@
  * Formats data for the `/return-requirements/{sessionId}/check-your-answers` page
  * @module CheckYourAnswersPresenter
  */
-
+const { formatLongDate } = require('../base.presenter.js')
```

Comment:
Just some whitespace

```suggestion

const { formatLongDate } = require('../base.presenter.js')

```